pkg/controller/install/webhook.go

```diff
@@ -163,6 +168,7 @@ func (i *StrategyDeploymentInstaller) createOrUpdateValidatingWebhook(ogNamespac
 }
 
 const WebhookDescKey = "webhookDescriptionGenerateName"
+const WebhookHashKey = "webhookDescriptionSHA"
```

Comment:
We should probably align with the naming convention for `pod-template-hash` (see the dep hash label too)

Reply:
Understood, I'll make the following update:
```
const WebhookHashKey = "olm.webhook-description-hash"
```

Reply:
Updated in the latest version of the PR.